packages/ui/src/app/pages/Proposals/PastProposals.tsx

```diff
@@ -21,7 +23,10 @@ export const PastProposals = () => {
   const [filters, setFilters] = useState(ProposalEmptyFilter)
 
   const { types, stages } = usePastProposals()
-  const { isLoading, proposals, pagination } = useProposals({ status: 'past', filters })
+
+  const { order, getSortProps } = useSort<ProposalOrderByInput>('createdAt')
```

Comment:
Oh! I forgot about this one also:
```suggestion
  const { order, getSortProps } = useSort<ProposalOrderByInput>('statusSetAtTime')
```

Reply:
@thesan pushed the requested changes.